pkg/tools/gen/testdata/doc/pkg/md_got/docs/main.md

```diff
@@ -0,0 +1,76 @@
+# main
```

Comment:
md_got/docs/main.md is the expect test result for the md/docs/main.md, you can copy the content of xmd_got/docs/main.md to md/docs/main.md and delete md_got/docs/main.md